charmtools/utils.py

```diff
@@ -520,3 +520,7 @@ def prefix_lines(lines, lineno):
                    m=message)
         i += 1
     return i == 0
+
+
+class BuildError(Exception):
```

Comment:
maybe move this to an exceptions module, it isn't really a utility, just a nitpick though